ztp/gitops-subscriptions/argocd/example/acmpolicygenerator/acm-group-du-sno-ranGen.yaml

```diff
@@ -34,24 +34,43 @@ policies:
     - path: source-crs/ClusterLogForwarder.yaml
       patches:
       - spec:
+          serviceAccount:
+            name: logcollector
           outputs:
-            - type: "kafka"
-              name: kafka-open
-              # below url is an example
-              url: tcp://10.46.55.190:9092/test
+            - name: default-lokistack
+              type: lokiStack
+              lokiStack:
+                target:
+                  name: logging-loki
+                  namespace: openshift-logging
+                authorization:
+                  token:
+                    from: serviceAccount
+              tls:
+                ca:
+                  key: service-ca.crt
+                  configMapName: openshift-service-ca.crt
           pipelines:
-          - inputRefs:
-            - audit
-            - infrastructure
-            labels:
-              label1: test1
-              label2: test2
-              label3: test3
-              label4: test4
-            name: all-to-default
+          - name: all-to-default
+            inputRefs:
+              - applications
+              - audit
+              - infrastructure
             outputRefs:
-            - kafka-open
-    - path: source-crs/ClusterLogging.yaml
```

Comment:
The preferred output format should be kafka unless this is no longer available

Reply:
Kafka is still supported, I checked with the CLO team. I thought it was removed with 6.0